src/test/java/net/sf/jabref/importer/fileformat/BiblioscapeImporterTest.java

```diff
@@ -0,0 +1,97 @@
+package net.sf.jabref.importer.fileformat;
+
+import java.io.ByteArrayInputStream;
+import java.io.IOException;
+import java.io.InputStream;
+import java.nio.charset.StandardCharsets;
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.List;
+
+import net.sf.jabref.Globals;
+import net.sf.jabref.JabRefPreferences;
+import net.sf.jabref.bibtex.BibtexEntryAssert;
+import net.sf.jabref.importer.OutputPrinterToNull;
+import net.sf.jabref.model.entry.BibEntry;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.junit.runners.Parameterized.Parameter;
+import org.junit.runners.Parameterized.Parameters;
+
+@RunWith(Parameterized.class)
+public class BiblioscapeImporterTest {
+
+    private BiblioscapeImporter bsImporter;
+
+    @Parameter(value = 0)
+    public String biblioscapeType;
+
+    @Parameter(value = 1)
+    public String expectedBibType;
+
+
+    @Parameters
+    public static Collection<String[]> types() {
+        return Arrays.asList(new String[][] {
+            {"journal", "article"},
+                {"book section", "inbook"},
+                {"book", "book"},
+                {"conference", "inproceedings"},
+                {"proceedings", "inproceedings"},
+                {"report", "techreport"},
+                {"master thesis", "mastersthesis"},
+                {"thesis", "phdthesis"},
+        });
+    }
+
+    @Before
+    public void setUp() throws Exception {
+        Globals.prefs = JabRefPreferences.getInstance();
+        bsImporter = new BiblioscapeImporter();
+    }
+
+    @Test
+    public void testGetFormatName() {
+        BiblioscapeImporter importer = new BiblioscapeImporter();
+        Assert.assertEquals(importer.getFormatName(), "Biblioscape");
+    }
+
+    @Test
+    public void testGetCLIID() {
```

Comment:
I think, this test does not depend on the parameters. Please separate parameterized tests from non-parameterized.


Maybe move the parameterized ones to `BiblioscapeImporterTestTypes.java`.